move/seal-policy/sources/seal_policy.move

```diff
@@ -0,0 +1,28 @@
+// Copyright (c), Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module app::seal_policy;
+
+use enclave::enclave::Enclave;
+use sui::hash::blake2b256;
+use app::weather::WEATHER;
+
+const ENoAccess: u64 = 0;
+
```

Comment:
let's add a comment here and in https://github.com/MystenLabs/nautilus/blob/main/move/enclave/sources/enclave.move
that we do not check that the latest version of the enclave is used, but if needed, one can also take EnclaveConfig as an argument and check config_version